src/components/m-accordion-item/js/accordion-item.js

```diff
@@ -0,0 +1,162 @@
+import UiEvents from '../../../js/ui-events';
+import on from '../../../js/on';
+import { requestAnimationFrame } from '../../../js/request-animation-frame';
+import {
+  add,
+  remove,
+} from '../../../js/class-list';
+import {
+  publish,
+  subscribe,
+} from '../../../js/pubsub';
+
+class AccordionItem extends UiEvents {
+  static DEFAULTS = {
+    container: '.js-accordion-item',
```

Comment:
The `container` class is only need if the container node for UIEvents is not equal with the web component's root node itself.
In your case you can just unset it and remove the `.js-accordion-item` class.